src/sass/index.sass

```diff
@@ -123,20 +126,22 @@
   #pause
     display: none
   
+  #menu
+    top: auto !important
+
   .window-app
     display: flex !important
     flex-direction: column !important
     width: 100% !important
     min-width: 0 !important
     min-height: 0 !important
-    top: 0 !important
+    top: auto !important
     left: 0 !important
     height: auto !important
-    max-height: calc(100vh - var(--hotbar-height)) !important
+    max-height: 70vh !important
```

Comment:
I set the max height of popups to 70% of the screen width so extremly long popups don't fill the whole screen, I think it looks better this way